Fix GCC detection since it can be e.g. x86_64-pc-linux-gnu-gcc

--- configure.ac
+++ configure.ac
@@ -1102,7 +1102,7 @@
 AC_MSG_CHECKING(if symbol visibility is supported)
 visibility_cflags=""
 case `basename $CC` in 
-  gcc*)
+  *gcc*)
      dnl Determine version number (watch out: the outside square
      dnl parenteses are vitally important here!)
      gccversion=["` ${CC} --version | head -1 | ${SED} 's/.*\([0-9]\.[0-9]\.[0-9]*\).*/\1/' `"]
